src/Microsoft.AspNet.DataProtection/DataProtectionEventId.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNet.DataProtection
+{
+    public enum DataProtectionEventId
+    {
+        KeyServices = 1,
+        XmlKeyManager,
+        KeyRingProvider,
+        CertificateXmlEncryptor,
+        DpapiNGXmlEncryptor,
+        DpapiNGXmlDecryptor,
```

Comment:
Another con for enums: if you remove an item, all those that follow will get a different value